test/regress/recursion-issue.txt

```diff
@@ -0,0 +1,23 @@
+;;; TOOL: run-interp-spec
+;;; NOTE: ref: issue-2398
+(module
+  (type $out-i32 (func (result i32)))
+
+  (table funcref
+    (elem
+      $runaway-with-return
+    )
+  )
+
+  (func $runaway-with-return (export "runaway-with-return") (type $out-i32) (result i32)
```

Comment:
This function seems to have both `type` and `result`.   Is that intended and somehow part of the repro case?

Reply:
reverse engineering/wasm2wat artifact :p

also we just realized we can minify this further